android-components/components/support/base/src/main/java/mozilla/components/support/base/log/sink/TestModeLogSink.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.base.log.sink
+
+import mozilla.components.support.base.log.Log
+
+/**
+ * <code>LogSink</code> implementation that prints to console.
+ */
+internal class TestModeLogSink : LogSink {
+
+    override fun log(priority: Log.Priority, tag: String?, throwable: Throwable?, message: String) {
+        val printMessage = StringBuilder()
+        printMessage.append(priority.name[0])
+        printMessage.append(" ")
```

Comment:
```suggestion
        printMessage.append("$priority.name[0] ")
```
Could we combine the 2?

Reply:
Yeah we can, was initially intending to keep the printing logic the same but we can certainly improve it. Simplified the implementation with `buildString {}` and combining these 2 in one.